.gitignore

```diff
@@ -19,3 +19,6 @@ local.settings.json
 .env
 
 /bin
+
+# ignore JetBrains IntelliJ config
+**.idea
```

Comment:
@glindstr This belongs in the global user-space `.gitignore` on your local machine. Adding it there will prevent you from having issues with projects in general.

Reply:
No problem, I'll remove.